doc/release-notes.rst

```diff
@@ -2,16 +2,12 @@
  Release Notes
 ===============
 
-v12.1.4 Luminous (RC)
-=====================
-
-This is the fifth release candidate for Luminous, the next long term stable
-release.
+v12.2.0 Luminous
+================
 
-Ceph Luminous (v12.2.0) will be the foundation for the next long-term
-stable release series.  There have been major changes since Kraken
-(v11.2.z) and Jewel (v10.2.z), and the upgrade process is non-trivial.
-Please read these release notes carefully.
+This is the first release of Luminous, the current long term stable release.
```

Comment:
This is the first release of the Luminous v12.2.z long term stable release series.

(avoid saying "current" or it will be confusing later)

Reply:
ack, makes sense